corehq/apps/app_manager/templates/app_manager/partials/add_new_module_modal.html

```diff
@@ -0,0 +1,101 @@
+{% load i18n %}
+
+<div class="modal fade add-new-module-modal" id="add-new-module-modal" tabindex="-1" role="dialog" aria-labelledby="add-new-module">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="modal-title">{% trans "Create a New Menu" %}</h4>
+        </div>
+        <div class="modal-body modal-flex-container">
+            <button type="button" class="new-module-option" data-type="survey">
+                <div class="panel panel-default">
+                    <div class="panel-heading">{% trans "Surveys" %}</div>
+                    <div class="panel-body">
+                        <i class="fa-regular fa-file"></i>
+                        <p>{% blocktrans %}Collect data once{% endblocktrans %}</p>
+                    </div>
+                </div>
```

Comment:
can you make this panel into a reusable partial so that there is less duplication?

Reply:
Refactored in 55b8ebd539ee325211cb90da4644111499b44166